specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/network/rfc7540/connection.management/server.sent.100k.message/client.rpt

```diff
@@ -0,0 +1,183 @@
+#
```

Comment:
These large message scenarios feel like they belong in `flow.control`, not `connection.management`.

Reply:
Done